codeforamerica/vita-min
# Fyst 1088 move financial transaction to shared xml other cleanup

## Link to pivotal/JIRA issue
- https://codeforamerica.atlassian.net/browse/FYST-1088
## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
- Moved the financial transaction out into a shared folder and then modified XML to include it.
- Also includes a fix for a broken tax refund form
- Stubs for refunds have been changed to a positive value to allow for it to show up

## How to test?
- For each of ID, NC, and MD step through the process off creating a tax return
- At the end, fill out the routing number and account number and mark direct deposit. Note that these cannot match
- After clicking through, click on the xml and near the bottom it should show the `FinancialTransaction` element.

## Screenshots (for visual changes)
![image](https://github.com/user-attachments/assets/03eb16cd-bc58-4ce7-91af-224e870b5fd1)
